tutorials/pretrain_openwebtext.md

```diff
@@ -24,7 +24,10 @@ The script will take about 15 min to run.
 
 
 ## Pretraining
-
+You might need to install huggingface datasets
+```bash
+pip install datasets
+```
```

Comment:
Hi! Where do the pretraining scripts use `datasets`? It's only in the openwebtext preparation script https://github.com/Lightning-AI/lit-gpt/blob/main/scripts/prepare_openwebtext.py#L24 but this is in a different stage.

I think you meant to put this in the snippet in line 17